src/index.ts

```diff
@@ -118,6 +125,28 @@ export default class NewRelicLambdaLayerPlugin {
     await Promise.all(promises);
   }
 
+  public async checkIntegrationRole() {
+    const { accountId } = this.config;
+    if (!accountId) {
+      this.serverless.cli.log(
+        "No newRelic accountId specified; Cannot check for required NewRelicLambdaIntegrationRole."
+      );
+      return;
+    }
+
+    try {
+      const params = {
+        RoleName: `NewRelicLambdaIntegrationRole_${accountId}`
```

Comment:
Do we know if the integration role should remain accountId or be converted to trustedAccountKey?

Reply:
We may want to stay with accountId throughout, unless there's a specific requirement for the format of the IAM role  ARN to change.

Reply:
This appears to be consistent with how the CLI behavior. Where in the README are you seeing language that suggested a trusted account key is  preferable to an account id? We may want to clarify that language as that is not the case. The trusted account key is used for subaccounts, where the `accountId` would be the subaccount and the `trustedAccountKey` is the parent account.